lib/Doctrine/ORM/Mapping/MappingException.php

```diff
@@ -941,4 +944,17 @@ public static function invalidUniqueConstraintConfiguration($className, $indexNa
             )
         );
     }
+
+    /**
+     * @param mixed $givenValue
+     * @return self
+     */
+    public static function invalidOverrideType(string $expectdType, $givenValue)
+    {
+        return new self(sprintf(
+            'Expected %s, but %s was given.',
+            $expectdType,
+            is_object($givenValue) ? get_class($givenValue) : gettype($givenValue)
```

Comment:
```suggestion
            get_debug_type($givenValue)
```

Since we have the polyfill